google-account-plugin/src/com/google/cloud/tools/intellij/login/IntellijGoogleLoginService.java

```diff
@@ -169,6 +169,7 @@ public void logIn(@Nullable final String message,
       @Nullable final IGoogleLoginCompletedCallback callback) {
     UsageTrackerProvider
         .getInstance().trackEvent(LoginTracking.CATEGORY, LoginTracking.LOGIN, "login.start", null);
+    final CredentialedUser activeUser = users.getActiveUser();
     users.removeActiveUser();
```

Comment:
Then attempting to add an account when already logged in will not work: to initiate the auth flow the user has to be logged out, otherwise the flow will be skipped.